src/Illuminate/Support/Str.php

```diff
@@ -2083,4 +2083,17 @@ public static function flushCache()
         static::$camelCache = [];
         static::$studlyCache = [];
     }
+
+    /**
+     * Split a string by a given separator.
+     *
+     * @param  string  $separator
```

Comment:
> [!Note]
> `explode()` will now throw [`ValueError`](https://www.php.net/manual/en/class.valueerror.php) when `separator` parameter is given an empty string (`""`).
Therefore, we might narrow this to [`non-empty-string`](https://phpstan.org/writing-php-code/phpdoc-types#other-advanced-string-types)
```suggestion
     * @param  non-empty-string  $separator
```